dev/travis/script.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Copyright © 2015 Magento. All rights reserved.
+# See COPYING.txt for license details.
+
+set -e
+export PATH="./../../../vendor/bin:$PATH"
+
+case $TEST_SUITE in
+    unit)
+        cd dev/tests/unit
+        phpunit
+        ;;
+    integration)
+        cd dev/tests/integration
+        phpunit
+        ;;
+    static)
+        cd dev/tests/static
+        phpunit --filter 'Magento\\Test\\Php\\LiveCodeTest::(testCodeStyle|testAnnotationStandard)'
```

Comment:
Actually @mazhalai is already working on fixing static tests. Example can be found here - https://travis-ci.org/mazhalai/magento2/builds/115163946#L746 and https://travis-ci.org/mazhalai/magento2/jobs/115362517#L594 . But thanks for the help :)
It looks pretty fast now, but it may take a lot of time for phpmd in case of changes across whole code base (rare case, but still may happen).
I don't insist, though I don't see a big benefit from merging these tests together.